app/webpacker/styles/sections/hero.scss

```diff
@@ -275,3 +275,121 @@ $mobile-cutoff: 800px;
     padding-bottom: 2em;
   }
 }
+
+
+// New styles for LandingHeroComponent
+.landing-hero {
+	display: flex;
+	flex-direction: column;
+	justify-content: center;
+	margin: auto;
+	width: 100%;
+	max-width: 2000px;
+	max-height: fit-content;
+
+	@include mq($from: tablet) {
+		max-height: 420px;
+		flex-direction: row-reverse;
+	}
+
+	// 50% width on child items '.content' and 'picture' (100% on mobile)
+	& > * {
+		width: 100%;
+
+		@include mq($from: tablet) {
+			width: 50%;
+		}
+	}
+
+	.content {
+		align-self: center;
+		box-sizing: border-box;
+		z-index: 1;
+		padding: 20px;
+
+		@include mq($from: tablet) {
+			padding: 50px;
+		}
+
+		@media only screen and (min-width: 1200px) {
+			padding: 90px;
+		}
+
+		// If no image use this
+		&.wide {
+			padding: 60px 0;
+			width: 100%;
+			
+			@include mq($from: tablet) {
+				width: 80%;	
+			}
+	
+			h1.title {
+				margin-top: 0;
+			}
+		}
+
+		h1.title {
+			font-size: 54px;
```

Comment:
Is this font size a bit big on mobile? Works on desktop though so happy for you to leave as is - up to you @jenhadfield-dfe 

![image](https://github.com/DFE-Digital/get-into-teaching-app/assets/87642394/8e31f814-5e2b-4988-9cdb-9394a64942ec)

Reply:
Creates a few instances where we have a very short word hanging on its own eg

![image](https://github.com/DFE-Digital/get-into-teaching-app/assets/87642394/75daf238-34f7-4063-9817-2b03717f7517)